doc/storage.md

```diff
@@ -10,11 +10,14 @@ discourse: 7735
 About storage </explanation/storage>
 Create a storage pool <howto/storage_create_pool>
 Create an instance in a pool <howto/storage_create_instance>
-Add a storage volume <howto/storage_add_volume>
-Configure storage settings <howto/storage_configure>
-List pools and volumes <howto/storage_list>
-Move or copy volumes <howto/storage_move>
-Back up storage volumes <howto/storage_backup>
-Resize storage <howto/storage_resize>
+Create a storage volume <howto/storage_create_volume>
+Configure storage pool <howto/storage_configure_pool>
+Configure storage volume <howto/storage_configure_volume>
```

Comment:
Should we go with "<action> a storage volume" or "<action> storage volumes" type phrasing on this menu, as its a mixture a the moment.

Reply:
I prefer `<action> a storage volume/pool`, except for `List storage pools/volumes` (since you don't list just one ;) ). 
But we need the "a" in there. If that makes the title too long, we should drop the "storage".